Reword admonitions for switching SCC accounts (#3514)
theforeman/foreman-documentation: guides/common/modules/proc_adding-an-scc-account.adoc

Use the following procedure to add your SCC account to {Project}.
To use the CLI instead of the {ProjectWebUI}, see the xref:cli-Adding_an_SCC_Account_to_Server_{context}[].

[WARNING]
====
The SCC Manager plugin does not support multiple SCC accounts within one organization.

> An additional SCC account should only be added for the purpose of switching SCC accounts.

Reviewer: ```suggestion
Add an additional SCC account only for the purpose of switching SCC accounts.
```
or perhaps:
```suggestion
Add an additional SCC account only if you want to switch to using another SCC account.
```

Author: Thanks, applied the second suggestion.

> If you need to use additional SCC accounts, you need to add them to other organizations.

Reviewer: ```suggestion
If you need to use additional SCC accounts, add them to other organizations.
```

Author: Applied.
